### ISSUE TITLE
Preferences slow to open

### ISSUE THREAD

**Issue by jacksarick** [wavebox/waveboxapp#8]
Mac OSX version El Capitain 10.11.6
Wavebox version 3.1.2

Preferences page very slow to open, taking up to 20-30 seconds to open up each time. Tested across multiple multiple machines, all have similar issues, even when Wavebox is the only application open.

**Comment by Thomas101** [comment]
Hi, thanks for reporting. Would you be able to give me the following info...

* How are you opening the settings? (Bottom left corner, App Menu > Wavebox > Preferences or `Cmd+,`)
* If you press the add mailbox button how long does that take to launch?
* How many mailboxes have you added
* If you press the Apple icon in top left of your mac window and then go to About this Mac. Could you send me a screenshot of that window?

Thanks!

**Comment by Thomas101** [comment]
Hi, we've been doing some digging around here and managed to reproduce it. Looks like it's down to  a call that checks if Wavebox starts on system startup. The call to `app.getLoginItemSettings()` seems to hang on osx. I raised an issue a while ago about this https://github.com/electron/electron/issues/7447 but it's down to osx.

Probably the best thing to do here is try a different library. There's [node-auto-launch](https://github.com/Teamwork/node-auto-launch) which looks like it works differently. I'll give that one a try and see if performance improves

**Comment by jacksarick** [comment]
No matter how I open it, it takes the same amount of time.

Add mailbox is instantaneous, until I get to the settings page, then it lags for a bit. Opening sub-windows works perfectly fine for everything except the settings page.

3 mailboxes. 2 Google, 1 Office 365

System info:

```
OS X El Capitan
Version 10.11.6 (15G1004)
MacBook Air (13-inch, Early 2015)
Processor: 1.6 GHz Intel Core i5
Memory: 8 GB 1600 MHz DDR3
Graphics: Intel HD Graphics 6000 1536 MB
```


From my experience, I don't think it has anything to do with opening the window, because other actions open windows without delay. That would mean that it is something in the window that takes a while to fetch. I think I might play around with it and see if I can narrow down what is taking the most time.

**Comment by Thomas101** [comment]
We found that it hangs if we have a network drive setup to connect on system startup that's not reachable. Do you have any in your startup items?

**Comment by Thomas101** [comment]
Hi a fix for this has just been release in 3.1.3. Wavebox will autoupdate shortly or if you want to get it sooner you can either re-launch Wavebox to trigger an autoupdate or redownload from https://wavebox.io/download Thanks!
